eva-accession-core/src/main/java/uk/ac/ebi/eva/accession/core/ClusteredVariantAccessioningService.java

```diff
@@ -17,23 +17,94 @@
  */
 package uk.ac.ebi.eva.accession.core;
 
+import uk.ac.ebi.ampt2d.commons.accession.core.AccessioningService;
 import uk.ac.ebi.ampt2d.commons.accession.core.BasicAccessioningService;
 import uk.ac.ebi.ampt2d.commons.accession.core.DatabaseService;
+import uk.ac.ebi.ampt2d.commons.accession.core.exceptions.AccessionCouldNotBeGeneratedException;
+import uk.ac.ebi.ampt2d.commons.accession.core.exceptions.AccessionDeprecatedException;
+import uk.ac.ebi.ampt2d.commons.accession.core.exceptions.AccessionDoesNotExistException;
+import uk.ac.ebi.ampt2d.commons.accession.core.exceptions.AccessionMergedException;
+import uk.ac.ebi.ampt2d.commons.accession.core.exceptions.HashAlreadyExistsException;
+import uk.ac.ebi.ampt2d.commons.accession.core.models.AccessionVersionsWrapper;
+import uk.ac.ebi.ampt2d.commons.accession.core.models.AccessionWrapper;
 import uk.ac.ebi.ampt2d.commons.accession.hashing.SHA1HashingFunction;
 
 import uk.ac.ebi.eva.accession.core.persistence.DbsnpMonotonicAccessionGenerator;
+import uk.ac.ebi.eva.accession.core.service.DbsnpClusteredVariantMonotonicAccessioningService;
 import uk.ac.ebi.eva.accession.core.summary.ClusteredVariantSummaryFunction;
 
+import java.util.Collections;
+import java.util.List;
+import java.util.stream.Collectors;
+
 /**
  * Service entry point for accessioning and querying clustered variants.
- *
+ * <p>
  * TODO Support EVA clustered variants, see {@link SubmittedVariantAccessioningService} for reference
  */
-public class ClusteredVariantAccessioningService extends BasicAccessioningService<IClusteredVariant, String, Long> {
+public class ClusteredVariantAccessioningService implements AccessioningService<IClusteredVariant, String, Long> {
```

Comment:
what does these method implementations add over `BasicAccessioningService`? just having the fork point for dbsnp and eva clustered variant accessioning services? I would prefer waiting to do this until we have both  services ready, because maybe we can reuse SubmittedVariantAccessioningService and make it `EvaAndDbsnpVariantAccessioningService<VARIANT_MODEL>` or similar

Reply:
it had to do with the `getByHash` but since it was removed this was reverted back to extend BasicAccessioningService. Agree that it will be better to wait until we have both services ready